modules/spring-config/src/main/java/org/mule/config/spring/SpringXmlConfigurationBuilder.java

```diff
@@ -132,6 +128,7 @@ else if (domainContext != null)
         // some beans may try to look up other beans via the Registry during
         // preInstantiateSingletons().
         muleContext.addRegistry(registry);
```

Comment:
Same comment as before.  We're deprecating the multiple registries, but this configurtion builder uses it.